bin.src/build-high-resolution-hips-qg

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# This file is part of pipe_tasks.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""A custom script to generate a QuantumGraph for GatherResourceUsageTask.
+See ``GatherResourceUsageTask.build_quantum_graph`` for more information
+(including why this script exists), and use ``--help`` for documentation on
+arguments.
+"""
+
+import sys
+
+from lsst.pipe.tasks.hips import HighResolutionHipsTask
+
+
+if __name__ == "__main__":
+    HighResolutionHipsTask.build_quantum_graph_cli(sys.argv[1:])
```

Comment:
I understand that this implementation is a temporary measure to get us through DP0.2. In the future it seems like it should work like `butler` and be a dynamic pluggable command for `pipetask` such that `pipetask hips segment` works. @erykoff can you make a ticket for that and maybe @n8pease can take a look. As things stand the current implementations can't auto-generate any sphinx documentation because there is no entry point for sphinx to examine.

Reply:
I'm not sure if the ticket should be to make this particular usage work in `pipetask` or should be to get the actual quantum graph generation code to be able to figure out healpix overlaps with tracts/patches?

Reply:
Treating it as a private API is fine for a while but do we know how long @TallJimbo thinks it's going to be until this code disappears?

Reply:
I've added a description of how to use `build-high-resolution-hips-qg` to the task documentation.  I assume that's a net good, but I'm not sure how to note that this is a temporary workaround.